bash_completion.d/000_bash_completion_compat.bash

```diff
@@ -29,6 +29,11 @@ _comp_deprecate_func 2.12 _modules _comp_compgen_kernel_modules
 _comp_deprecate_func 2.12 _installed_modules _comp_compgen_inserted_kernel_modules
 _comp_deprecate_func 2.12 _usergroup _comp_compgen_usergroup
 _comp_deprecate_func 2.12 _complete_as_root _comp_as_root
+_comp_deprecate_func 2.12 _included_ssh_config_files _comp__included_ssh_config_files
```

Comment:
This is being renamed/made to a private function, so exposing the deprecated name is not necessary.
```suggestion
```